abci/example/kvstore/kvstore.go

```diff
@@ -138,19 +138,18 @@ func (app *Application) Info(context.Context, *types.InfoRequest) (*types.InfoRe
 		}
 	}
 
-	var defaultLanePriority uint32
-	if len(app.lanes) > 0 {
-		defaultLanePriority = app.lanes[defaultLane]
+	defLane := ""
+	if len(app.lanes) != 0 {
+		defLane = "default"
```

Comment:
Should this be a `const`?

Reply:
Agree, it should be a const. It appears in many places.